### DISCUSSION TITLE
求大佬帮忙！阿里系列小程序编译后都是白屏，TypeError: taro.defineComponent is not a function

### DISCUSSION THREAD

**Discussion by abelshare** [NervJS/taro#16275] [Q & A]
阿里系的小程序，按官方的项目初始化后，编译启用小程序都是白屏  TypeError: taro.defineComponent is not a function

![image](https://github.com/user-attachments/assets/3d3fc320-8f22-4e97-9bb1-5f6d73780837)

![image](https://github.com/user-attachments/assets/4d755407-2216-4aff-9f9c-672f7c6e306f)

**Comment by abelshare** [comment]
用的vite taro & NutUI

**Comment by TheKonka** [comment]
https://github.com/NervJS/taro/issues/16271#issuecomment-2277589549
试下这个看看

**Comment by zmqcherish** [comment]
taro init 选项 ： vue3 + ts + sass + yarn + vite + github(最新) + vue3-NutUI，yarn安装依赖直接报错，解决后 yarn dev:weapp 启动，也报错。taro真难用... 
![image](https://github.com/user-attachments/assets/a124d77d-9d39-4c5b-b277-50ee4f5b9503)

**Reply by asharca** [reply]
一摸一样的错误，请问解决了吗

**Reply by gy1001** [reply]
使用vite报错， 使用webpack可以
